app/components/Search.vue

```diff
@@ -0,0 +1,59 @@
+<!-- TODO: 
+  Die Suche soll Schrittweise erweitert werden:
+  1. Suche nach Biertiteln. In den Suchergebnissen werden nur Biere angezeigt.
+  2. Erweiterung auf Brauereien. Ergebnisse sollen strukturiert angezeigt werden. (Divider: Biere, Brauereien, etc.)
+  3. Erweiterung auf Biersorten.
+  [4. Erweiterung auf Volltext in allen Artikeln]
+-->
+
+<template android>
+  <Page actionBarHidden="true">
+      <StackLayout backgroundColor="#3c495e">
+    <SearchBar hint="Search hint" v-model="searchQuery" @textChange="onTextChanged" @submit="onSubmit" />
+    <ActivityIndicator busy="true" @busyChange="onBusyChanged" />
+    <Label>Suchergebnisse für die Anfrage: {{searchQuery}} </Label>
```

Comment:
Let's use localization from the start. Adding that later is probably much more work than now

Reply:
Solve it later but not to late #9